src/test/scala/com/sky/kafka/configurator/TopicConfigurationParserSpec.scala

```diff
@@ -116,4 +116,39 @@ class TopicConfigurationParserSpec extends BaseSpec {
     TopicConfigurationParser(new StringReader(yml)).right.value shouldBe List.empty
   }
 
+  it should "parse topics configured using yaml anchors and aliases" in {
```

Comment:
Feel like the second test covers this as well. Maybe just keep the second test?

Reply:
I agree, its unnecessary duplication. It would be great to have this as a property based test, but I'm not sure we use those in this project.

Reply:
Yeah happy to remove it from here. Do we think testing anchors/aliases at a higher level would be useful (e.g. `KafkaConfiguratorIntSpec`) or is showing that the parsing is working here enough?

Reply:
It could have value only as documentation, showing that we support it.

Reply:
I've removed this test and added on to `KafkaConfiguratorIntSpec` in 75222fb. Also updated the readme to mention that anchors can be used.